enterprise/ha/src/main/java/org/neo4j/kernel/impl/ha/ClusterManager.java

```diff
@@ -132,7 +145,7 @@
     private final StoreDirInitializer storeDirInitializer;
     LifeSupport life;
 
-    public ClusterManager( Provider clustersProvider, File root, Map<String,String> commonConfig,
+    public ClusterManager( Provider clustersProvider, File root, Map<String,IntFunction<String>> commonConfig,
```

Comment:
A bigger change though. If we split common config from per-instance config we won't have to make this change.